css/styles.css

```
@@ -142,15 +156,15 @@ h2 {
     border-radius: 50%;
     background-color: #3498db;
     border: 2px solid white;
-    box-shadow: 0 0 3px rgba(0,0,0,0.5);
+    box-shadow: 0 0 3px rgba(0, 0, 0, 0.5);
     position: relative;
 }
 
 .resize-icon:before {
-    content: "↘";
+    content: "↗";
     position: absolute;
     top: -1px;
     left: 2px;
     font-size: 14px;
     color: white;
-}
+}
```

See the changes below to see where I extract this from.
```suggestion
}

#location-upload-panel {
   position: absolute;
   top: 10px;
   right: 10px;
   z-index: 1000;
   background-color: white;
   padding: 15px;
   border-radius: 5px;
   box-shadow: 0 0 10px rgba(0,0,0,0.1);
}

#error-message {
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    background: white;
    padding: 15px;
    border-radius: 5px;
    box-shadow: 0 0 10px rgba(0,0,0,0.1);
    z-index: 1000;
    max-width: 80%;
    text-align: left;
}
```